core/cardano-addresses.cabal

```diff
@@ -53,8 +53,8 @@ library
       NoImplicitPrelude
   ghc-options: -Wall -Wcompat -fwarn-redundant-constraints
   build-depends:
-      aeson
-    , base >=4.7 && <5
+      base >=4.7 && <5
+    , aeson ^>= 1.5.6.0 || ^>= 2.0.3.0
```

Comment:
For this library, I don't think we have any particular need to support both aeson versions.

Reply:
We have quite a lot of code that relies on `aeson-1`.  I think it will be less disruptive to downstream developers if we support both versions on a very temporary basis.